Library/Formula/csshx.rb

```diff
@@ -1,13 +1,11 @@
-require 'formula'
-
 class Csshx < Formula
-  homepage 'http://code.google.com/p/csshx/'
-  url 'https://csshx.googlecode.com/files/csshX-0.74.tgz'
-  sha1 'aa686b71161d6144d539d077b960da10d7b96993'
+  homepage "https://github.com/brockgr/csshx"
+  url "https://csshx.googlecode.com/files/csshX-0.74.tgz"
+  sha256 "eaa9e52727c8b28dedc87398ed33ffa2340d6d0f3ea9d261749c715cb7a0e9c8"
 
-  head 'https://code.google.com/p/csshx/', :using => :git
+  head "https://github.com/brockgr/csshx.git"
 
   def install
-    bin.install 'csshX'
+    bin.install "csshX"
   end
 end
```

Comment:
Can a test be added to do something more substantial than e.g. `--version` or `--help`? See `cmake.rb` for an example of an application formula with a good test and `tinyxml2.rb` for an example of a library formula with a good test. Thanks!